main.py

```
@@ -697,6 +748,103 @@ def analyze(project_path, scan_all, commit, interactive, verbose, export_json, e
             inc_analyzer.save_hash(inc_analyzer.compute_project_hash())
             if verbose:
                 click.echo("   Saved incremental cache")
+        
+        # Quality Check (Phase 1 - Analyzer Agent)
+        if quality_check:
+            from src.ai.content_analyzer import ContentAnalyzer
+            from rich.console import Console
+            from rich.table import Table
+            
+            if verbose:
+                click.echo(f"\n📊 Running quality analysis...")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)